### ISSUE TITLE
Applet Bug

### ISSUE THREAD

**Issue by tcarver01** [linuxmint/cinnamon-spices-applets#5386]
### Applet version/Build date

3.3.1

### Cinnamon version

6.0.4

### Distribution

Mint 21.3

### Graphics hardware *and* driver used

AMD Cezanne vendor: Gigabyte driver: amdgpu v: kernel pcie: speed: 8 GT/s lanes: 16     ports: active: HDMI-A-1 empty: DP-1 bus-ID: 05:00.0 chip-ID: 1002:1638 class-ID: 0300   Display: x11 server: X.Org v: 1.21.1.4 driver: X: loaded: amdgpu,ati     unloaded: fbdev,modesetting,vesa gpu: amdgpu display-ID: :0 screens: 1

### Applet name and maintainer

anime-wallpaper@flyflyan @fly206

### What happened?

After suspend or reboot the weather applet does not load with data. I can force the applet to load data by going to configure/more options/reload weather. No errors reported.

### Other information

[lwzk.txt](https://github.com/linuxmint/cinnamon-spices-applets/files/13958183/lwzk.txt)

**Comment by fly206** [comment]
Is this bug about the applet Weather (weather@mockturtl)？

**Comment by tcarver01** [comment]
Yes it was. The bug has been fixed since then and working nicely.
